### DISCUSSION TITLE
Found: workaround for array multiplication in constraints & objfcn using MILPModel

### DISCUSSION THREAD

**Discussion by datadrivensupplychain** [dirkschumacher/ompr#348] [Show and tell]
Hello,

I figured out a way to use 3D and 4D arrays in constraints and the objective function using MILPModel.  I am using this for my work in supply chain network optimization, and wanted to share with the community.  OMPR is awesome!!

https://github.com/datadrivensupplychain/teaching_bits/blob/main/ompr_milpmodel_3d_4d_array_multiplication.r

Ralph Asher
ralph@datadrivensupplychain.com

**Comment by dirkschumacher** [comment]
Thanks! I move this to the discussions
